### ISSUE TITLE
glibc: Include patch to fix issues with Chinese characters (sort -u or uniq)

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-core#47]
Reported as Bug 13063 at sourceware.org, https://sourceware.org/bugzilla/show_bug.cgi?id=13063. As suggested in comments, a patch was included for Bug 17563 (https://sourceware.org/bugzilla/show_bug.cgi?id=17563) - which was described to be tested with Fedora - and working.

Attempt to apply this patch and check back.

Test case from @Icenowy : 

```bash
printf "%s\n" ，。 | uniq
```

**Comment by gumblex** [comment]
Could test this by generating a list of all Unicode characters and feeding into sort.

**Comment by MingcongBai** [comment]
Seemingly fixed with Glibc 2.25 as the test case no longer returns incorrect output.

```
root@general [ patches@master ] # printf "%s\n" ，。 | uniq
，。
```

And also, with the original test case, which contains the following content (too long, using pastebin):

https://pastebin.aosc.io/view/ad7fcca9

And the test result as follows (content contained in `test.txt`):

```
root@general [ patches@master ] # cat test.txt | sort -u
㑗
冧
壪
揻
撶
溋
熕
茾
```
